### DISCUSSION TITLE
Trekarta has extremly high battery use

### DISCUSSION THREAD

**Discussion by paulle** [andreynovikov/trekarta#153] [Q&A]
Trekarta has extremly high battery use. Is this normal? Sony xperia xz2, android9.
![screenshot_20190131-170037](https://user-images.githubusercontent.com/4852079/52122670-040ab700-2624-11e9-949f-226497c5e0cf.png)

**Comment by andreynovikov** [comment]
Is it at normal operation or when loading and importing maps?

**Comment by paulle** [comment]
It's at normal operation. Map was downloaded days before and it's locally stored.

Am 4. Februar 2019 19:05:05 MEZ schrieb Andrey Novikov <notifications@github.com>:
>Is it at normal operation or when loading and importing maps?
>
>-- 
>You are receiving this because you authored the thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/andreynovikov/trekarta/issues/62#issuecomment-460349902

-- 
K-9 Mail

**Comment by andreynovikov** [comment]
What exactly are you doing in application? Writing track? Navigating? Something else?

**Comment by paulle** [comment]
I was writing a track. And as far as I remember I did nothing special. 
In these days I have to stay at home, but in one of the next days, I can repeat what I did, if this will help you.

**Comment by paulle** [comment]
It seems the problem is solved.Why and how I don't know.
I deleted trekarta and installed otherwise (all via f-droid).
Later I wrote a track (about 40 minutes) and battery usage was 2%.
That's much better than before (16min 10%).

**Comment by datenbank-projekt** [comment]
Hi,

I am using the latest version available from F-Droid and the problem seems still to exist.
If I use Trekarta (really great app!) just to display my position on a map from time to time I can live with the battery usage. A couple of days ago, I did something different (to my normal use): I let Trekarte track me and also follow me on the map (left two buttons). The battery usage was around 55%.
I did exactly the same with Wikiloc ( https://play.google.com/store/apps/details?id=com.wikiloc.wikilocandroid ) - track and show position - and that had only about 20% battery usage.
I also used GPSLogger ( https://f-droid.org/de/packages/com.mendhak.gpslogger ) (all three at the same time) and had that set to get position every 3 seconds. GPSLogger used only 0.5% of the battery.

With this much energy use Trekarte has great maps an featuers but when it comes to longtime use it's very "problematic" - to say the least.

Anyways - again - I really like the app.

**Comment by andreynovikov** [comment]
I began development of Trekarta mostly with motor vehicles in mind where power source is always available. Trekarta is designed to record every single GPS location, that's why it uses so much power when recording. Recently I focused on hiking and cycling where power consumption is of great importance. I plan to implement periodic location acquisition in foreseeable future.

**Comment by datenbank-projekt** [comment]
Hi andrey,

thanks for the quick reply.
For the periodic acquisition, maybe take a look on GPSLogger. The recording there can be finetuned a lot:  time periods of acquisition and points to be ignored (when one is not moving) ...
Thanks for keeping the development up.

**Comment by AgentSteel** [comment]
Hello, I also used Trekarta (version 2023.03.22 fdroid) to record my hiking track, and yes, I can confirm it consumes a lot of battery.
